src/internal.c

```diff
@@ -4070,6 +4071,7 @@ static INLINE int Decrypt(WOLFSSH* ssh, byte* plain, const byte* input,
             break;
 
         case ID_AES128_CBC:
+            sz = sz - (sz % AES_BLOCK_SIZE); /* decrypt AES block size */
```

Comment:
Am reviewing this code section. It could be redundant or may be better to be an error case if it gets to this point and the buffer is not a multiple of AES block size.